dev-libs/openssl/openssl-1.0.2f.recipe

```diff
@@ -40,6 +40,11 @@ REQUIRES="
 	lib:libz$secondaryArchSuffix >= 1.2.3
 	ca_root_certificates
 	"
+if [ -n "$secondaryArchSuffix" ]; then
+	REQUIRES="$REQUIRES
+		openssl == $portVersion base
+		"
+fi
```

Comment:
What I had noticed was that the recipes of `1.0.0t` and `1.0.1h` both had this conditional append for their REQUIRES. So I wrote [in the first comment of this PR](#issuecomment-176677702) that I was not sure wether this change was required/desirable for 1.0.2f or not. From your comment I understand that this conditional append was removed on purpose for the 1.0.2 series. So I just removed it from 1.0.2f. Should I also remove it from 1.0.0t and 1.0.1r ? Or is it safer to just keep that part unchanged?